a11y-dialog.js

```diff
@@ -399,4 +399,4 @@
   } else if (typeof global === 'object') {
     global.A11yDialog = A11yDialog;
   }
-}(this));
+}(typeof global !== "undefined" ? global : window));
```

Comment:
I know this is awkward to ask for that, but could you use single quotes so that semistandard linting is respected? Once done, I’m supper happy to merge and release. :)